website/docs/advanced-concepts/granular-access-control/reference/permissions.mdx

```diff
@@ -0,0 +1,717 @@
+        </tr>
+        <tr>
+          <td>&nbsp;&nbsp;&nbsp;Default Roles</td>
+          <td>Associate Role</td>
+          <td>Allows viewing and assigning the default role to users or groups.</td>
+        </tr>
+        <tr>
+          <td>&nbsp;&nbsp;&nbsp;&nbsp;Custom &nbsp;&nbsp;&nbsp;&nbsp;Roles</td>
+          <td>Associate Role</td>
+          <td>Allows viewing and assigning the given custom role to users or groups.</td>
+        </tr>
+      </tbody>
+    </table>
+  </TabItem>
+  <TabItem label="Others" value="view-others">
+    <table className="permissions-table" style={{ width: '100%', display: 'table', marginLeft: '2px' }}>
+      <thead>
+        <tr>
+          <th>Resource</th>
+          <th>View also grants below permissions</th>
+          <th>Permission Impact</th>
+        </tr>
+      </thead>
+      <tbody>
+        <tr>
+          <td><b>Audit logs</b></td>
+          <td>NA</td>
+          <td>Allows viewing the Appsmith instance audit logs that helps in monitoring the instance.</td>
+        </tr>
+      </tbody>
+    </table>
+  </TabItem>
+</Tabs>
+
+## Execute permission
+
+The **Execute** permission permits users to execute actions such as running queries in Appsmith. When the **Execute** permission is assigned to a parent resource, it's also inherited by the child resources.
+
+<Tabs queryString="gac-resource">
+  <TabItem label="Application Resources" value="execute-application-resources">
+    <table className="permissions-table" style={{ width: '100%', display: 'table', marginLeft: '2px' }}>
+      <thead>
+        <tr>
+          <th>Resource</th>
+          <th>Execute also grants below permissions</th>
+          <th>Permission Impact</th>
+        </tr>
+      </thead>
+      <tbody>
+        <tr>
+          <td><b>Workspace</b></td>
+          <td>NA</td>
+          <td>Allows running queries within the given workspace.</td>
+        </tr>
+        <tr>
+          <td><b>Application</b></td>
+          <td>NA</td>
+          <td>Allows running queries within the given application.</td>
+        </tr>
+        <tr>
+          <td><b>Page</b></td>
+          <td>NA</td>
+          <td>Allows running queries within the given application.</td>
+        </tr>
+      </tbody>
+    </table>
+  </TabItem>
+  <TabItem label="Datasources & Environments" value="execute-datasources-and-environments">
+    <table className="permissions-table" style={{ width: '100%', display: 'table', marginLeft: '2px' }}>
+      <thead>
+        <tr>
+          <th>Resource</th>
+          <th>Execute also grants below permissions</th>
+          <th>Permission Impact</th>
+        </tr>
+      </thead>
+      <tbody>
+        <tr>
+          <td><b>Datasources</b></td>
+          <td>NA</td>
+          <td>Allows running queries in all datasources within the workspace.</td>
+        </tr>
+        <tr>
+          <td>&nbsp;&nbsp;&nbsp;&nbsp; &nbsp;Application &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Datasource</td>
```

Comment:
Sorry, no such thing as an application datasource. It's just a datasource used in the application. Datasources are always workspace wide.

Reply:
Yes - a naming typo.